MyPlayground.playground/Contents.swift

```diff
@@ -0,0 +1,78 @@
+func Mean(numbers: [Int]) -> Double {
+    
+    if numbers.isEmpty { return 0 }
+    
+    var sumOfNumbers = 0
+    
+    for elementOfNumber in numbers {
+        sumOfNumbers += elementOfNumber
+    }
+    let mean = Double(sumOfNumbers) / Double(numbers.count)
+    
+    return mean
+}
+
+
+func Median(numbers: [Int]) -> Double {
+    
+    if numbers.isEmpty { return 0 }
+    
+    let numbersSorted = numbers.sorted()
+    
+    if numbersSorted.count % 2 == 0 {
+        let lowerBoundOfInterval = numbersSorted.count / 2 - 1
+        let upperBoundOfInterval = numbersSorted.count / 2
+        let median = (Double(numbersSorted[lowerBoundOfInterval]) + Double(numbersSorted[upperBoundOfInterval])) / 2
+        
+        return median
+    } else {
+        
+        return Double(numbersSorted.count / 2)
+    }
+}
+
+
+func Moda(numbers: [Int]) -> Set<Int> {
+    
+    if numbers.isEmpty { return Set<Int>() }
+    
+    var valueOfItem = 0
+    var previousItem = 0
+    var count = [Int: Int]()
+    var moda = Set<Int>()
+    let numbersSorted = numbers.sorted()
+    
+    for ItemOfNumber in numbersSorted {
```

Comment:
Capitalized letters:
```suggestion
    for itemOfNumber in numbersSorted {
```

Reply:
Moreover:
```suggestion
    for number in numbersSorted {
```

First, be simple.
Second, How you can translate "item of number" into Russian?

Reply:
itemOfNumbers - значение номера. Corrected to number.

Reply:
Fixed.